app/models/odk/response_parser.rb

```diff
@@ -178,7 +178,7 @@ def lookup_and_check_form(params)
 
       # if form version is outdated, error
       if form.current_version.code != params[:version] &&
-          form.current_version.sequence.to_s != params[:version]
+          form.current_version.number != params[:version]
```

Comment:
Can you explain the context behind [issue 8831](https://redmine.sassafras.coop/issues/8831) if you remember? Maybe this second check is obsolete now.

Reply:
This will all be changing soon anyway. But yes you can just remove the second check.